app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,375 @@
+    <string name="send_send_label">Spendi i miei bei Moneroj</string>
+    <string name="send_send_timed_label">Spendi i miei bei Moneroj (%1$s)</string>
+    <string name="send_really_send_hint">Ultimi passo - Conferma!</string>
+    <string name="send_qr_invalid">Non è un codice QR</string>
+    <string name="send_qr_address_invalid">Non è un codice QR pagamento valido</string>
+    <string name="send_address_invalid">Non è un indirizzo valido</string>
+    <string name="send_preparing_progress">Preparando la transazione</string>
+    <string name="send_title">Invia</string>
+    <string name="send_available">Saldo: %1$s XMR</string>
+    <string name="send_address_title">Indirizzo</string>
+    <string name="send_amount_title">Ammontare</string>
+    <string name="send_settings_title">Impostazioni</string>
+    <string name="send_confirm_title">Conferma</string>
+    <string name="send_success_title">Fatto</string>
+    <string name="send_summary_heading">Riassunto</string>
+    <string name="send_amount_too_large">Ammontare > Fondi</string>
+
+    <string name="send_amount_label">Ammontare</string>
+    <string name="send_fee_btc_label">Commissione (XMR)</string>
+    <string name="send_fee_label">Commissione</string>
+    <string name="send_dust_label">Polvere</string>
+    <string name="send_total_btc_label">Totale (XMR)</string>
+    <string name="send_total_label">Totale</string>
+
+    <string name="send_amount">%1$s XMR</string>
+    <string name="send_fee">+%1$s di commissione</string>
+
+    <string name="send_create_tx_error_title">Errore nella creazione della transazione</string>
+
+    <string name="tx_list_fee_pending">incl. %1$s di commissione</string>
+    <string name="tx_list_fee">- Commissione %1$s</string>
+    <string name="tx_list_amount_failed">(%1$s)</string>
+    <string name="tx_list_failed_text">fallita</string>
+    <string name="tx_list_amount_negative">- %1$s</string>
+    <string name="tx_list_amount_positive">+ %1$s</string>
+
+    <string name="tx_status_btc">Status</string>
+    <string name="tx_address">Indirizzo</string>
+    <string name="tx_timestamp">Timestamp</string>
+    <string name="tx_id">ID TX</string>
+    <string name="tx_id_btc">ID TX\n(BTC)</string>
+    <string name="tx_key">Chiave TX</string>
+    <string name="tx_destination">Destinazione</string>
+    <string name="tx_destination_btc">Destinazione\n(BTC)</string>
+    <string name="tx_paymentId">ID Pagamento</string>
+    <string name="tx_blockheight">Blocco</string>
+    <string name="tx_amount">Ammontare</string>
+    <string name="tx_amount_btc">Ammontare\n(BTC)</string>
+    <string name="tx_fee">Commissione</string>
+    <string name="tx_transfers">Trasferimenti</string>
+    <string name="tx_notes">Note</string>
+    <string name="tx_button_copy">Copia negli appunti</string>
+    <string name="tx_copy_message">Copiata negli appunti</string>
+    <string name="tx_copy_label">Dettagli della transazione</string>
```

Comment:
maybe just `dettagli transazione`. let's keep the text as short as possible

Reply:
Don't know why this, the one above and some of the next three reviews ar flagged as obsolete, but i don't see any changes oin the code.

Reply:
Agreed. The problem is that github hides some conversations. Dunno why. Sorry.